pkg/storage/plan_data_exchange_approachStore.go

```diff
@@ -0,0 +1,75 @@
+package storage
+
+import (
+	_ "embed"
+	"fmt"
+
+	"github.com/lib/pq"
+
+	"github.com/cms-enterprise/mint-app/pkg/sqlqueries"
+
+	"github.com/google/uuid"
+	"go.uber.org/zap"
+
+	"github.com/cms-enterprise/mint-app/pkg/models"
+	"github.com/cms-enterprise/mint-app/pkg/shared/utilitysql"
+	"github.com/cms-enterprise/mint-app/pkg/shared/utilityuuid"
+	"github.com/cms-enterprise/mint-app/pkg/sqlutils"
+)
+
+// PlanDataExchangeApproachCreate creates a new plan data exchange approach
+func PlanDataExchangeApproachCreate(np sqlutils.NamedPreparer, _ *zap.Logger, approach *models.PlanDataExchangeApproach) (*models.PlanDataExchangeApproach, error) {
```

Comment:
this function could still use the sqlutils.GetProcedure

Reply:
8e37d2c50